src/metabase/public_settings.clj

```diff
@@ -323,18 +322,6 @@
   :setter     :none
   :getter     driver.u/available-drivers-info)
 
-(defsetting types
-  "Field types"
-  :visibility :public
-  :setter     :none
-  :getter     (fn [] (types/types->parents :type/*)))
-
-(defsetting entities
-  "Entity types"
-  :visibility :public
-  :setter     :none
-  :getter     (fn [] (types/types->parents :entity/*)))
```

Comment:
I guess we were already not using this?

Reply:
Yeah, I could not find any place that was using this. Removing it didn't affect anything at least